tests/integration/fileTests.py

```diff
@@ -449,9 +449,9 @@ def test_file_generation(self):
 
         self.assertEqual(response.status_code, 200)
         json = response.json
-        self.assertEqual(json["status"], "waiting")
+        self.assertIn(json["status"], ["waiting","finished"])
         self.assertEqual(json["file_type"], "D1")
```

Comment:
A small thing, but the docstring says that this is testing the D2 file generation.